std/format/internal/read.d

```diff
@@ -12,15 +12,20 @@
  */
 module std.format.internal.read;
 
-import std.range.primitives;
-import std.traits;
+import std.range.primitives : ElementEncodingType, ElementType, isInputRange;
 
-import std.format;
+import std.traits : isAggregateType, isArray, isAssociativeArray,
+    isDynamicArray, isFloatingPoint, isIntegral, isSomeChar, isSomeString,
+    isStaticArray, StringTypeOf;
 
-package(std.format) void skipData(Range, Char)(ref Range input, scope const ref FormatSpec!Char spec)
+import std.format : FormatSpec;
+
+package(std.format) void skipData(Range, Char)(ref Range input,
+                                               scope const ref FormatSpec!Char spec)
```

Comment:
When you find yourself starting a line in column 48 (!), that's a sure sign something has gone wrong, and you need to re-think the decisions that lead you to this outcome.

If you want to break this line, I think a better place to do so is at the start of the function arguments:
```suggestion
package(std.format) void skipData(Range, Char)(
    ref Range input, scope const ref FormatSpec!Char spec)
```
Or if you prefer each argument on its own line:
```suggestion
package(std.format) void skipData(Range, Char)(
    ref Range input,
    scope const ref FormatSpec!Char spec)
```